src/xform_to_datalog/testdata/simple_auth_logic.dl

```diff
@@ -0,0 +1,6 @@
+.decl says_cond1(x0: symbol, x1: symbol)
+.decl grounded_dummy(x0: symbol)
+.decl says_fact1(x0: symbol, x1: symbol)
+says_fact1("prin1", thing_x) :- says_cond1("prin1", thing_x).
+says_cond1("prin1", "foo").
+grounded_dummy("dummy_var").

```

Comment:
Similar question as above. This looks like the output of your other test. Is this meant to be checked in?

Reply:
Yes, this is used in a test as a golden file.